.github/workflows/handler-test.yml

```
@@ -18,13 +20,29 @@ jobs:
         id: vars
         run: echo ::set-output name=run-url::https://github.com/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID
 
-      # Checkout the branch of the pull request under test
+      # Checkout the branch of the pull request under tests/
```

This comment was referring to "the pull request under test" like "the thing being tested", not the `/tests` directory.

Perhaps this would clear it up?

```suggestion
      # Checkout the branch of the pull request being tested
```